worker/worker.go

```diff
@@ -1614,6 +1617,15 @@ func (w *Worker) UploadMultipartUploadPart(ctx context.Context, r io.Reader, buc
 	}, nil
 }
 
+func (w *Worker) initAccounts(refillInterval time.Duration) (err error) {
+	if w.accounts != nil {
+		panic("priceTables already initialized") // developer error
+	}
+	keyPath := fmt.Sprintf("accounts/%s", w.id)
```

Comment:
Do we want to add some type of check on whether all workers in a cluster are using unique worker ids? Currently this is not the case and the worker ID defaults to `worker` which 'll likely end up in a situation where multiple workers are using the same accounts.

Reply:
Not sure we can do a lot of checking but I updated the config to require a worker id to be set manually if connecting to a remote bus. Otherwise it defaults to `worker`.